src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,32 @@
+package duke;
```

Comment:
Perhaps duke.exception as a package name would be clearer? Especially if the number of custom exceptions increase.